bin/utils/convert-argv.js

```diff
@@ -128,28 +128,32 @@ module.exports = function(...args) {
 	}
 
 	if (!configFileLoaded) {
-		return processConfiguredOptions({});
+		return processConfiguredOptions(null);
 	} else if (options.length === 1) {
 		return processConfiguredOptions(options[0]);
 	} else {
 		return processConfiguredOptions(options);
 	}
 
 	function processConfiguredOptions(options) {
-		const webpackConfigurationValidationErrors = validateSchema(webpackConfigurationSchema, options);
-		if (webpackConfigurationValidationErrors.length) {
-			const error = new WebpackOptionsValidationError(webpackConfigurationValidationErrors);
-			console.error(error.message, `\nReceived: ${typeof options} : ${JSON.stringify(options, null, 2)}`);
-			process.exit(-1); // eslint-disable-line
+		if (options) {
+			const webpackConfigurationValidationErrors = validateSchema(webpackConfigurationSchema, options);
+			if (webpackConfigurationValidationErrors.length) {
+				const error = new WebpackOptionsValidationError(webpackConfigurationValidationErrors);
+				console.error(error.message, `\nReceived: ${typeof options} : ${JSON.stringify(options, null, 2)}`);
+				process.exit(-1); // eslint-disable-line
+			}
+		} else {
+			options = {};
```

Comment:
Nothing wrong with this code, but I usually avoid mutations like this one.

Reply:
Yea, agreed. I think we would have needed a larger refactor to process args without validating options, as explained a bit further down